stable/appmesh-prometheus/README.md

```diff
@@ -21,6 +21,29 @@ helm upgrade -i appmesh-prometheus eks/appmesh-prometheus \
 --namespace appmesh-system
 ```
 
+Enable EBS CSI Driver:  
```

Comment:
Also I wonder if this belongs in the prerequisites section. Maybe something about "To use a persistent volume for storing data you'll need the EBS CSI Driver installed. This isn't necessarily part of installing this chart, since it's one time setup for the cluster.

Reply:
Thank you for the notificaiton, Joe. I just made some research, PVC is not mandatory for Prometheus to work, but highly recommended. Without a PVC, Prometheus data will be stored on the local filesystem of the container in which it is running. If the container is restarted or recreated, the data will be lost. This is not ideal for a time-series database that needs to store data over a long period of time. Because the original text already emphasizes the need to use persistent volume, I will add the Optional label to this section for further notice.